insights/parsers/bdi_read_ahead_kb.py

```diff
@@ -0,0 +1,48 @@
+"""
+BDIReadAheadKB - file ``/sys/class/bdi/*/read_ahead_kb``
+========================================================
+
+This parser reads the content of ``/sys/class/bdi/*/read_ahead_kb``.
+"""
+
+from insights import Parser, parser
+from insights.specs import Specs
+from insights.parsers import ParseException
+
+
+@parser(Specs.bdi_read_ahead_kb)
+class BDIReadAheadKB(Parser):
+    """
+    Class ``BDIReadAheadKB`` parses the content of the ``/sys/class/bdi/*/read_ahead_kb``.
+
+    Attributes:
+        read_ahead_kb (int): value of read_ahead_kb.
+
+    Raises:
+        ParseException: When content is empty or unparseable
+
+    A typical sample of the content of this file looks like::
+
+        128
+
+    Examples:
+        >>> type(bdi_read_ahead_kb)
+        <class 'insights.parsers.bdi_read_ahead_kb.BDIReadAheadKB'>
+        >>> bdi_read_ahead_kb.read_ahead_kb
+        128
+    """
+
+    def parse_content(self, content):
+        if len(content) != 1:
+            raise ParseException("Error: ", content[0] if content else 'empty file')
```

Comment:
@JoySnow  First thing, shouldn't there be a .format on the string to add the text to the exception string like "Error: {0}".format(content)? Second thing, wouldn't it be better to do a ", " .join(content) instead of only printing the first entry in the list?

Reply:
Thanks for the reminding.